find_intersecting_spherical_cones.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+def find_intersection(hashmap1, hashmap2):
+    '''Checks the condensed dictionaries hashmap1 and hashmap2 for spherical cones that both face each 
+    other and intersect.
+    
+    A new hashmap, intersecting_spheres, is returned with keys from hashmap1 and values from hashmap2 
+    '''
+    intersecting_spheres = {}
+    for sphere1, values1 in hashmap1.items(): # looping through every sphere(key)-value pair in hashmap1
```

Comment:
values1 and values2 contain apex coordinates and perpendicular vector coordinates for each sphere respectively such that: 
sphere : [ (apex_x, apex_y, apex_z) , (perp_x, perp_y, perp_z) ] 

Explain what is stored in values in your function docstring above